gensrc/thrift/BackendService.thrift

```diff
@@ -161,4 +161,5 @@ service BackendService {
 
     TStreamLoadRecordResult get_stream_load_record(1: i64 last_stream_record_time);
 
+    oneway void clean_trash();
```

Comment:
Is it safe to be called multi times using `oneway`?
And is the method `start_trash_sweep()` thread safe?